app/routes/land-business-details/confirm-details.js

```diff
@@ -1,34 +1,49 @@
 const cache = require('../../cache')
-const { getLandCovers } = require('../../api/crown-hosting/land-cover')
 const getMapParcels = require('../../map')
+const joi = require('joi')
 
 module.exports = [
   {
     method: 'GET',
     path: '/confirm-details',
     options: {
       handler: async (request, h) => {
-        const agreement = await cache.get('agreement', request.yar.id)
-        const application = agreement.application
-        const { totalHectares, landCovers } = await getLandCovers(application.selectedOrganisation.organisationId, application.callerId)
         const mapParcels = await getMapParcels(request)
 
         if (!mapParcels.parcels) {
           return h.view('no-response')
         }
 
-        return h.view('land-business-details/confirm-details',
-          {
-            apiKey: mapParcels.apiKey,
-            parcels: mapParcels.parcels,
-            center: mapParcels.center,
-            mapStyle: mapParcels.mapStyle,
-            sbi: mapParcels.sbi,
-            name: application.selectedOrganisation.name,
-            address: application.selectedOrganisation.address,
-            totalHa: totalHectares,
-            landCovers
+        return h.view('land-business-details/confirm-details', mapParcels)
+      }
+    }
+  },
+  {
+    method: 'POST',
+    path: '/confirm-details',
+    options: {
+      validate: {
+        payload: joi.object({
+          landControlCheck: joi.boolean().required(),
```

Comment:
I'm not sure this name fits.  `landControlCheck` doesn't relate to the question.  Asking them if they have management control of the land is the next question.  This question is about whether the land data is correct.

What do you think about renaming it to maybe `isLandCorrect` or `landCorrect`?